misc/python/materialize/cloudtest/k8s/testdrive.py

```diff
@@ -35,7 +37,11 @@ def __init__(self, release_mode: bool, aws_region: Optional[str] = None) -> None
 
         container = V1Container(
             name="testdrive",
-            image=self.image("testdrive", release_mode=release_mode),
+            # NOTE(benesch): is it intentional that this does not pass through
```

Comment:
We write testdrive fragments with the assumption that they will always be run on the latest testdrive. So none of the mzcompose workflows allow for a custom testdrive. Doe that answer your question?